src/pages/Dashboard/Dashboard.component.tsx

```
@@ -18,6 +23,25 @@ export const Dashboard = () => {
             >
                 <SalesChart salesData={salesData} />
             </SectionContainer>
+            <Stack direction={isDesktop ? 'row' : 'column'} gap={4}>
+                <Box {...(isDesktop && { minWidth: spacing(95) })}>
+                    <SectionContainer variant='md' heading='Latest Customers'>
+                        <AppList variant='user' list={users}></AppList>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)